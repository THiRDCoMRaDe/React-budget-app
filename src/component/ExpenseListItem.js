import React from 'react';
import { Link } from 'react-router-dom';

const ExpenseListItem = ({description, amount, createdAt, id}) => (
    <>
        <Link to={`/edit/${id}`}>
            <h3>
                description: {description}
            </h3>
        </Link>
        <p>
            {amount} - {createdAt}
        </p>

    </>
);


export default ExpenseListItem;
//export default ExpenseListItem;