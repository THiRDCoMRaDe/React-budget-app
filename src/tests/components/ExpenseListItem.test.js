import React from 'react';
import { shallow } from "enzyme";
import ExpenseListItem from "../../component/ExpenseListItem";
import expenses from "../fixtures/expenses";
import {expect} from "@jest/globals";

test('should render ExpenseListItem correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});