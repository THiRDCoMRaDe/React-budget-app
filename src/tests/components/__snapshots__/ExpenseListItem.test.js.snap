// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<Fragment>
  <Link
    to="/edit/1"
  >
    <h3>
      description: 
      Gum
    </h3>
  </Link>
  <p>
    195
     - 
    0
  </p>
</Fragment>
`;
